example/client/gateway/filter/README.md

```diff
@@ -0,0 +1,73 @@
+# How to use this example
+After launching the k8s cluster, do the following steps at the root directory.
+
+1. Deploy Vald cluster with filter-gateway
+```bash
+vim example/helm/values.yaml
+---
+...
+gateway:
+...
+    filter:
+        enabled: true
+...
+agent:
+    ngt:
+        dimension: 784
+        distance_type: l2
+...
+
+# deploy vald cluster
+helm repo add vald https://vald.vdaas.org/charts
+helm install vald vald/vald --values example/helm/values.yaml
+```
```

Comment:
```suggestion

      ```bash
      vim example/helm/values.yaml
      ---
      ...
      gateway:
      ...
      filter:
          enabled: true
      ...
      agent:
          ngt:
              dimension: 784
              distance_type: l2
      ...
      
      # deploy vald cluster
      helm repo add vald https://vald.vdaas.org/charts
      helm install vald vald/vald --values example/helm/values.yaml
      ```
```

Reply:
Thank you for the suggestion 🙏 
I will apply these changes!